modules/fundamental/src/ai/endpoints/mod.rs

```diff
@@ -1,16 +1,22 @@
 #[cfg(test)]
 mod test;
 
+use crate::ai::model::{Conversation, ConversationSummary};
```

Comment:
I would be great if you could group the `use` statements.

Reply:
Can you add that then?

Reply:
I personally don't mind it being ungrouped, my IDE automatically adds them that way.  I also don't know how to get the format checker to enforce that.

Reply:
Looks like https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#imports_granularity is what we would want.  but it's only in unstable right now.

Reply:
If we can't automatically fix it for them /w cargo fmt, we should not be nitpicking about it in PR reviews.  This is all just stylistic right?